packages/lib-user/src/components/shared/AllProjects/AllProjects.js

```diff
@@ -13,59 +19,109 @@ const StyledBox = styled(Box)`
   row-gap: 20px;
 `
 
-function AllProjects({ error, loading, projects }) {
+function AllProjects({
+  containerError = {},
```

Comment:
This is truthy, so your component will always show an error message unless you explicitly pass in `containerError={null}`.

Reply:
Oops, thank you for the catch!